### ISSUE TITLE
ManifestPlugin ignores paths.appAssetsManifest

### ISSUE THREAD

**Issue by andreffvalente** [jaredpalmer/razzle#1589]
### Current Behavior

Setting the razzle.config.js with the following:
```js
module.exports = {
  ...
  modifyPaths({ paths }) {
    paths.appAssetsManifest = `${paths.appBuild}/manifest.json`;

    return paths;
  }
  ...
};
```
ManifestPlugin ignores the path defined above and creates the file `assets.json` in the output directory.

### Expected behavior

ManifestPlugin to create the file defined in `paths.appAssetsManifest`.

### Suggested solution(s)

Change [this](https://github.com/jaredpalmer/razzle/blob/master/packages/razzle/config/createConfigAsync.js#L738) line to use `paths.appAssetsManifest`.

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Razzle           | 4.0.3
| Razzle Plugins   | 
| Node             |
| Browser          |
| npm/Yarn         |
| Operating System |
| TypeScript       | 
| React            |

**Comment by fivethreeo** [comment]
Will fix tomorrow

**Comment by fivethreeo** [comment]
Fixed in v4.0.4
